rails/db/migrate/20240726071341_create_active_storage_tables.active_storage.rb

```
@@ -0,0 +1,57 @@
+    create_table :active_storage_attachments, id: primary_key_type do |t|
+      t.string     :name,     null: false
+      t.references :record,   null: false, polymorphic: true, index: false, type: foreign_key_type
+      t.references :blob,     null: false, type: foreign_key_type
+
+      if connection.supports_datetime_with_precision?
+        t.datetime :created_at, precision: 6, null: false
+      else
+        t.datetime :created_at, null: false
+      end
+
+      t.index [ :record_type, :record_id, :name, :blob_id ], name: :index_active_storage_attachments_uniqueness, unique: true
```

rubocop

[convention] Do not use space inside array brackets.

(Layout/SpaceInsideArrayLiteralBrackets)